vellum/workflows/weekly_accounting/nodes/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 __all__ = [
     "BuildPersonalFinancialContext",
-    "GetCapitalOneTransactions", 
```

Comment:
Still call this the Capital One Transactions node, just use plaid under the hood to get the transactions